content/millennium/r4/encounters/encounter.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Encounter | R4 API
+---
+
+# Encounter
+
+* TOC
+{:toc}
+
+## Overview
+
+Placeholder
```

Comment:
update

Reply:
Added terminology bindings and overview, https://github.com/cerner/fhir.cerner.com/pull/238/commits/761f781e1eb58169b9b02d2530719b468b68a24d#diff-6590262e197180cc09f6ad70a2a01a5bL10